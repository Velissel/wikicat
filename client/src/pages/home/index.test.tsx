import React from 'react';
import { render, screen } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Home from './';

describe('home page', () => {
  it('should render', () => {
    const { container } = render(
      <BrowserRouter>
        <Home />
      </BrowserRouter>,
    );
    expect(screen.getByText('Catwiki Home')).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
