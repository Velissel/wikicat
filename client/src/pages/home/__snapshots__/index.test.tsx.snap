// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home page should render 1`] = `
<div>
  <div
    class="container"
  >
    <h1
      class="mt-5"
    >
      Catwiki Home
    </h1>
    <form
      action="#"
    >
      <div
        class="mb-3"
      >
        <label
          class="form-label"
          for="searchTerm"
        >
          Search cat by breed
        </label>
        <div
          class="input-group"
        >
          <input
            class="form-control"
            id="searchTerm"
            name="searchTerm"
            type="text"
            value=""
          />
          <button
            class="btn btn-primary"
            type="submit"
          >
            Search
          </button>
        </div>
      </div>
    </form>
    <h3
      class="mt-4"
    >
      Top 10 Searched Kitties
    </h3>
    To Be Implemented
  </div>
</div>
`;
